File: app/src/main/java/de/gaffga/android/zazentimer/bo/Session.java
package de.gaffga.android.zazentimer.bo;

public class Session {

    public String description;

    public int id;

    public String name;

    public Session() {
    }

    public Session(String str, String str2) {
        this.name = str;
        this.description = str2;
    }

    public String toString() {
        return this.name;
    }
}

File: app/src/main/kotlin/at/priv/graf/zazentimer/bo/Session.kt
package at.priv.graf.zazentimer.bo

data class Session(
    @JvmField var description: String? = null,
    @JvmField var id: Int = 0,
    @JvmField var name: String? = null,
    @JvmField var bellVolumes: List<SessionBellVolume> = emptyList(),
) {
    constructor(name: String, description: String) : this(
        description = description,
        id = 0,
        name = name,
    )
}
